integration-tests/contracts/ethereum_keeper_contracts.go

```diff
@@ -1662,6 +1718,32 @@ func (v *EthereumKeeperRegistrar) EncodeRegisterRequest(
 			amount,
 			common.HexToAddress(senderAddr),
 		)
+
+		if err != nil {
+			return nil, err
+		}
+		return req, nil
+	} else if v.registrar21 != nil {
+		registrarABI, err := abi.JSON(strings.NewReader(registrar21.AutomationRegistrarMetaData.ABI))
```

Comment:
don't need to do this, since we define the registrarABI at the top of the file :)

Reply:
sharp eyes 👀